target/linux/generic/pending-6.1/731-net-permit-ieee80211_ptr-even-with-no-CFG82111-suppo.patch

```diff
@@ -0,0 +1,68 @@
+From 686c603f67ae87bf21a61b5e4b1564443f41c3ee Mon Sep 17 00:00:00 2001
+From: Christian Marangi <ansuelsmth@gmail.com>
+Date: Thu, 20 Oct 2022 03:34:43 +0200
+Subject: [PATCH] net: permit ieee80211_ptr even with no CFG82111 support
```

Comment:
Hm, this should probably just be added to the backports so that `include/linux/netdevice.h` is backported as well

Reply:
Ughh, not really as `ieee80211_ptr` is used outside of backports as well

Reply:
it was a nice discovery... no idea if this will be accepted upstream...

Reply:
Well, from the upstream perspective I dont see why would they accept it as its guarded with a kernel symbol already.

Reply:
@hauke what to see a fun thing? check this patch....